book/art/fullnode-proposal.bob

```diff
@@ -0,0 +1,30 @@
+             .----------------------------------.
+             |  Fullnode                        |
+             |                                  |
+ .--------.  |  .------------------.            |
+ |        |---->|                  |            |
+ | Client |  |  | JSON RPC Service |            |
+ |        |<----|                  |            |
+ `----+---`  |  `------------------`            |
+      |      |     ^                            |
+      |      |     |    .----------------.      | .------------------.
+      |      |     |    | Gossip Service |<------>| Validators       |
+      |      |     |    `----------------`      | |                  |
+      |      |     |         ^                  | |                  |
+      |      |     |         |                  | |  .------------.  |
+      |      |  .--+---.  .--+---.  .---------. | |  |            |  |
+      |      |  | Bank |<-|Replay|  | Peer TX |<-----+ Upstream   |  |
```

Comment:
What not call "Peer TX Receiver" "Blob Fetch Stage"?

Reply:
In this proposal I started with BlobFetchStage. But, we already have a BlobFetchStage, and this block in the diagram also performs job of window service and retransmit stage.